docs/nor_gate_method.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: NOR gate method
+comments: true
+nav_order: 6
+---
+
+# NOR gate method to simplify boolean expressions
+{: .no_toc}
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## Introduction
+[NOR Gates](https://learn.circuitverse.org/docs/universal_gates.html#NOR-gate) are universal gates. By the virtue of functional completeness, NOR Gates can be used to fully represent a given boolean expression. This simplifies the expression such that only one standard gate is used throughout.  
```

Comment:
https://learn.circuitverse.org/docs/universal_gates.html#nor-gate
Please update the link for NOR gate.